src/components/learner-credit-management/data/hooks/useCancelContentAssignments.js

```diff
@@ -6,11 +6,13 @@ import { useToggle } from '@edx/paragon';
 import EnterpriseAccessApiService from '../../../../data/services/EnterpriseAccessApiService';
 import { learnerCreditManagementQueryKeys } from '../constants';
 import useBudgetId from './useBudgetId';
+import { applyFiltersToOptions } from './useBudgetContentAssignments';
 
 const useCancelContentAssignments = (
   assignmentConfigurationUuid,
   assignmentUuids,
   cancelAll = false,
+  tableFilters = [],
```

Comment:
[curious/clarification] When is it possible for `tableFilters` to be `undefined` to fallback to the empty array here? Will the `tableFilters` always be an empty list regardless if there are no filters actively applied?